Client/Frontend/Browser/TabManager.swift

```diff
@@ -380,8 +380,12 @@ class TabManager: NSObject {
     }
 
     func removeTabAndUpdateSelectedIndex(_ tab: Tab) {
-        removeTab(tab, flushToDisk: true, notify: true)
+        removeTab(tab, flushToDisk: true)
```

Comment:
No one called `notify: false`, but that isn't the only reason I removed it.  It is confusing API design to have the TabManager optionally call delegates, it should always call the delegates but pass along the requisite info so that they decide if they want to take action.

Reply:
👍